packages/amplication-data-service-generator/src/plugin-wrapper.ts

```diff
@@ -5,7 +5,7 @@ export type PluginWrapper = (
   func: (...args: any) => Module[] | Promise<Module[]>,
   event: EventNames,
   ...args: any
-) => any;
+) => Promise<Module[]>;
```

Comment:
please use a seperate PRs for changes like this for next time, we details about the reason behind the change for the record 

I understand why you did it, and it makes sense but I would love in cases like this to allow the original writer to review (in this case @barshimi )
It can be a mistake but it also may be on purpose